src/explore-education-statistics-common/src/services/types/problemDetails.ts

```diff
@@ -0,0 +1,17 @@
+export interface ProblemDetails {
+  title: string;
+  type: string;
+  status: number;
```

Comment:
Do we want to include `detail` on here for completeness?

Reply:
Or are we purposefully leaving it off for succinctness since we don't use it?

Reply:
I left it out for succinctness originally, but I've included it now.

To make things clearer for any consumers, I've also added docblocks for all the properties as well.